tests/quantization/test_unified_scales.py

```diff
@@ -561,7 +561,8 @@ def test_unified_scales_are_identical_in_onnx(tmp_path):
 
     with torch.no_grad():
         for quant_info in compression_ctrl.non_weight_quantizers.values():
-            quant_info.quantizer_module_ref.scale *= torch.abs(torch.rand_like(quant_info.quantizer_module_ref.scale))
+            scale = quant_info.quantizer_module_ref.scale
+            quant_info.quantizer_module_ref.scale = scale*torch.abs(torch.rand_like(scale))
```

Comment:
The idea behind the property usage was that all existing `.scale`-using code would still function properly with the change. Why didn't this line of code work?

Reply:
I already mentioned that torch.nn.Parameter does not work with property because torch.Module intercept this and raise error.  .scale=torch.nn.Parameter() will raise error that .scale already exist.